app/views/categories/index.html.haml

```diff
@@ -4,7 +4,7 @@
       %strong
       = link_to category.name, category_path(category)
     %br
-    - category.videos.each do |video|
+    - category.recent_videos.each do |video|
       %a{:href => video_path(video)}
```

Comment:
I would use `link_to` here just to remain consistent and you can as well use shorthand named routes `video` instead of `video_path(video)`.